src/main/java/software/amazon/event/kafkaconnector/EventBridgeSinkConfigValidator.java

```diff
@@ -142,6 +164,33 @@ private static void validateEndpointId(ConfigValue configValue) {
     validateValueWithPattern(configValue.name(), endpointId, arnPattern);
   }
 
+  private static void validateProfileName(ConfigValue configValue, EnvVarGetter envVarGetter) {
+    var profileName = (String) configValue.value();
+    // optional parameter
+    if (profileName == null || profileName.trim().isBlank()) {
+      return;
+    }
+
+    // throw if this config parameter and any AWS environment variables which overwrite its behavior
+    // (from DefaultCredentialsProvider chain) are set
+    var conflictingAwsEnvVars =
+        Stream.of(AWS_PROFILE, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN, AWS_ACCESS_KEY_ID)
+            .map(Enum::toString)
+            .filter(
+                env -> {
+                  return (envVarGetter.get(env) != null);
```

Comment:
Can be simplified to `env -> envVarGetter.get(env) != null`

Reply:
Done